app/views/welcome/index.html.erb

```diff
@@ -1,77 +1,123 @@
 <div class="parallax-container" id="index-banner">
-    <%= render "layouts/nav" %>
+  <%= render "layouts/nav" %>
+
   <div class="section no-pad-bot">
     <div class="container">
-    <br><br>
-      <h1 class="header center home_app_title">EventX</h1>
+      <br><br><br>
```

Comment:
@andela-megwali At some point in time, we will have to start doing this with css